public/components/integrations/components/add_integration_flyout.tsx

```diff
@@ -131,10 +131,16 @@ export function AddIntegrationFlyout(props: IntegrationFlyoutProps) {
   const fetchDataSourceMappings = async (
     targetDataSource: string
   ): Promise<{ [key: string]: { properties: any } } | null> => {
-    return fetch(`/api/console/proxy?path=${targetDataSource}/_mapping&method=GET`, {
-      method: 'POST',
-      headers: [['osd-xsrf', 'true']],
-    })
+    return fetch(
```

Comment:
todo item fetch should be replaced by osd's http client. compatibility is better